images/pre-commit/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM registry.ci.openshift.org/openshift/release:rhel-8-release-golang-1.18-openshift-4.12
```

Comment:
Not so important from CI perspective, since when we use a Dockerfile in Prow, the FROM is usally replaced with "pipeline:src" or any other base image